modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -217,8 +161,7 @@ include::{examplesdir}installation/lets_encrypt/renew.sh[]
 This script:
 
 * Renews all your issued certificates but does not upgrade Certbot.
-* In case you have enabled the post hook for your webserver in `cli.ini`, it will reload
-the web server configuration automatically if a certificate has been renewed.
+* In case you have enabled the post hook for your webserver in `cli.ini`, it will reload the web server configuration automatically if a certificate has been renewed.
```

Comment:
```suggestion
* In case you have enabled the post hook for your web server in `cli.ini`, it will reload the web server configuration automatically if a certificate has been renewed.
```